src/main/java/com/example/java/test/junior/developer/model/Category.java

```diff
@@ -14,11 +14,13 @@
 import javax.persistence.Table;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 
-@Data
+@Setter
+@Getter
```

Comment:
Why have you made these changes? What about equals/hashCode/toString? How it will work now without `@Data`?